### ISSUE TITLE
Multi-nodes support

### ISSUE THREAD

**Issue by binxuan** [sangmichaelxie/doremi#6]
Hi,

Thanks for sharing this opensource implementation. I am wondering does the current implementation support to train a larger reference model using multi-nodes?

Thanks

**Comment by sangmichaelxie** [comment]
No, we're only supporting single node training at the moment. I'll let you know if we do this in the future.
